### PR TITLE
feat: export controller event callback types

### PR DESCRIPTION
**Pull request by balloob**
This exports the controller event callback types so we can use the types in Z-Wave JS Server here https://github.com/zwave-js/zwave-js-server/blob/master/src/lib/forward.ts#L6

### CODE REVIEW

**Review by AlCalzone — CHANGES_REQUESTED** [review]
They should also be re-exported from `src/Controller.ts`, so I'm free to change the internal structure without breaking depending libs.

**Comment by balloob** [comment]
@zwave-js-bot fix lint

File: packages/zwave-js/src/lib/controller/Controller.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ interface ControllerEventCallbacks {
 	"heal network done": (result: ReadonlyMap<number, HealNodeStatus>) => void;
 }
 
-type ControllerEvents = Extract<keyof ControllerEventCallbacks, string>;
+export type ControllerEvents = Extract<keyof ControllerEventCallbacks, string>;
```
Comment (balloob) [inline packages/zwave-js/src/lib/controller/Controller.ts:126]: @AlCalzone any reason you prefer `Extract<keyof ControllerEventCallbacks, string>` over just `keyof ControllerEventCallbacks` ?

Comment (AlCalzone) [reply packages/zwave-js/src/lib/controller/Controller.ts:126]: I'm sure there was a reason for this 🤷🏻‍♂️
Or I copied it from somewhere where it was necessary and didn't remove it.

**Comment by AlCalzone** [comment]
@balloob can you write the fix-lint bot command again? I'm trying to debug why it is not working for other users.

**Comment by balloob** [comment]
@zwave-js-bot fix lint

**Comment by balloob** [comment]
(right now there is nothing to fix though)

**Comment by AlCalzone** [comment]
Okay, the bot thinks the comment is in an issue, not a PR, so it doesn't add the PR author to the list of authorized users. 👍🏻

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**